drivers/openstack/openstack.go

```diff
@@ -165,6 +166,11 @@ func GetCreateFlags() []cli.Flag {
 			Value: "",
 		},
 		cli.StringFlag{
+			Name:  "openstack-ip-pool-type",
```

Comment:
It would be better to have a flag to switch between neutron and nova-network not only for IP pool. We could use a boolean flag `--openstack-nova-network` with the default value to `false` as long as neutron is the mainstream networking service.